app/public/css/styles.css

```diff
@@ -1477,3 +1476,65 @@ p {
   background-position: center;
   background-image: url('../assets/input_pencil_icon.svg');
 }
+
+/* CARD */
+
+.card {
+  background-color: white;
+  padding: 40px 56px;
+  box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.08);
+}
+
+/* TRANSACTIONS */
+
+.transactions {
+  padding: 40px 56px;
+}
+
+.transactions-title {
+  margin-bottom: 24px;
+}
+
+.transaction-item {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  padding-top: 24px;
+  padding-bottom: 24px;
+  border-top: 1px solid var(--color-border);
+  color: var(--color-grey);
+}
+
+.transaction-date {
+  display: flex;
+  align-items: center;
+}
+
+.transaction-amount {
+  line-height: 32px;
+  font-size: 16px;
+  font-weight: bold;
+  text-align: right;
+}
+
+.transaction-amount.credit {
+  color: var(--color-success);
+}
+
+.transaction-amount.debit {
+  color: var(--color-error);
+}
+
+.transaction-address {
```

Comment:
Please fix this issue. You can misclick _View details_.
![screen recording 2019-01-15 at 12 48 05 2019-01-15 12_59_12](https://user-images.githubusercontent.com/2865988/51179303-62fecb00-18c5-11e9-9526-af3c42b3d578.gif)